### PR TITLE
Products

### PR DESCRIPTION
**Pull request by galganif**

### CODE REVIEW

File: src/tests/particles/products-test.ts [hunk 1]
```diff
@@ -69,7 +83,7 @@ describe('products test', () => {
             const verified = content.model && content.model.hasItems
                 && content.model.items['$template'].length > 0
                 && 1 === content.model.items.models.length;
-            if (verified) {
+            if (verified && !Flags.useNewStorageStack) {
```
Comment (shans) [inline src/tests/particles/products-test.ts]: I think we need to verify that the value is right when useNewStorageStack is set too.

Comment (galganif) [reply src/tests/particles/products-test.ts]: removed the check as discussed as I think it is redundant with the check in verifyFilteredBook. Maria can you check if this is ok?

File: src/tests/particles/artifacts/products-test-ng.recipes [hunk 2]
```diff
@@ -0,0 +1,58 @@
+// @license
```
Comment (mmandlis) [inline src/tests/particles/artifacts/products-test-ng.recipes]: nit - this file should be `.arcs` and upper camel case (ProductsTestNg.arcs) - i know this is inconsistent with the older file, but that one predates the convention :)

File: src/tests/particles/products-test.ts (hunk 1 above)
Comment (mmandlis) [reply src/tests/particles/products-test.ts]: the check you removed used to verify the rendered content, but that would become redundant anyway with the migration to uibroker, so it's ok to remove it. thanks for checking.

File: src/tests/particles/artifacts/products-test-ng.recipes (hunk 2 above)
Comment (galganif) [reply src/tests/particles/artifacts/products-test-ng.recipes]: Done

**Comment by galganif** [comment]
g3 presubmit in cl/288370074

**Pull request merged**